keyboards/handwired/onekey/keymaps/i2c_scanner/config.h

```diff
@@ -1,6 +1,11 @@
 #pragma once
 
 // AVR: can change to other supported values
-#define F_SCL 100000UL
+#ifdef __AVR__
+#   define F_SCL 100000UL
+#endif
 
-// TODO: add some default ARM configs for i2cv1 and i2cv2
+#ifdef PROTOCOL_CHIBIOS
+#   define I2C1_CLOCK_SPEED 100000
+#   define I2C1_DUTY_CYCLE STD_DUTY_CYCLE
+#endif
```

Comment:
these are the defaults for ChibiOS that support these defines. Not sur why they are needed.

Reply:
the avr define was already there - my thought/intent was to put this/these adjustment knob(s) in plain sight for someone using the scanner to do some hardware debugging -- "hidden" defaults have to first be known about, and then dug up :-P